examples/ocean_wind_mixing_and_convection.jl

```diff
@@ -28,62 +28,50 @@ using JLD2
 using Oceananigans
 using Oceananigans.Units: minute, minutes, hour
 
-# ## A vertically-stretched grid
+# ## The grid
 #
-# We use 32³ grid points with 2 meter grid spacing in the horizontal and
+# We use 32³ grid points with 2 m grid spacing in the horizontal and
 # varying spacing in the vertical, with higher resolution closer to the
-# surface. We use a two-parameter generating function to specify the
-# vertical cell interfaces:
+# surface,
 
-Nz = 24 # number of points in the vertical direction
-Lz = 32 # domain depth
-refinement = 1.2 # controls spacing near surface (higher means finer spaced)
-stretching = 8   # controls rate of stretching at bottom 
+σ = 1.1 # stretching factor
+Nz = 24
+Lz = 32
 
-## Normalized height ranging from 0 to 1
-h(k) = (k - 1) / Nz
-
-## Linear near-surface generator
-ζ₀(k) = 1 + (h(k) - 1) / refinement
-
-## Bottom-intensified stretching function 
-Σ(k) = (1 - exp(-stretching * h(k))) / (1 - exp(-stretching))
-
-## Generating function
-z_faces(k) = Lz * (ζ₀(k) * Σ(k) - 1)
+hyperbolically_spaced_faces(k) = - Lz * (1 - tanh(σ * (k - 1) / Nz) / tanh(σ))
 
 grid = VerticallyStretchedRectilinearGrid(size = (32, 32, Nz), 
                                           x = (0, 64),
                                           y = (0, 64),
-                                          z_faces = z_faces)
+                                          z_faces = hyperbolically_spaced_faces)
 
 # We plot vertical spacing versus depth to inspect the prescribed grid stretching:
 
 plot(grid.Δzᵃᵃᶜ[1:Nz], grid.zᵃᵃᶜ[1:Nz],
-     marker = :circle,
-     ylabel = "Depth (m)",
-     xlabel = "Vertical spacing (m)",
-     legend = nothing)
+      marker = :circle,
+      ylabel = "Depth (m)",
+      xlabel = "Vertical spacing (m)",
+      legend = nothing)
 
 # ## Buoyancy that depends on temperature and salinity
 #
 # We use the `SeawaterBuoyancy` model with a linear equation of state,
 
 buoyancy = SeawaterBuoyancy(equation_of_state=LinearEquationOfState(α=2e-4, β=8e-4))
 
-# where ``α`` and ``β`` are the thermal expansion and haline contraction
+# where $α$ and $β$ are the thermal expansion and haline contraction
 # coefficients for temperature and salinity.
 #
 # ## Boundary conditions
 #
 # We calculate the surface temperature flux associated with surface heating of
-# 200 W m⁻², reference density `ρₒ`, and heat capacity `cᴾ`,
+# 200 W m⁻², reference density `ρ`, and heat capacity `cᴾ`,
 
-Qʰ = 200  # W m⁻², surface heat flux
+Qʰ = 200  # W m⁻², surface _heat_ flux
```

Comment:
markdown formatting doesn't matter here... you still wanna leave that?

Reply:
I feel markdown formatting is literal, so emphasis is `_meaningful_` whether or not it renders italic.

Reply:
Fair enough. Just wanted to point out in case you were doing it out of markdown formatting habbit.

What does literal mean?

Reply:
By "literal" I mean it's not a "code". In pure text we have a few methods for emphasizing a word or phrase: `_emphasized_`, `*emphasized*`. Markdown compiles that syntax to a different style (eg italic or bold), but the meaning is the same. A non-literal syntax would be something like `\makebold(emphasis)`. `\makebold` is not literally emphasis; it's a command that only has meaning in the context of some imposed rule.

Reply:
OK, I get it. Agreed.

Reply:
This "literate" style is also used in the code. It's the difference between writing something like

```julia
# The initial condition is ``uᵢ(x) = U * cos(k x)``. We use a function to implement this initial condition:

function u_initial_condition(x)
    return U * cos(k * x)
end
```

and writing something like

```julia
# The initial condition is

uᵢ(x) = U * cos(k * x)
```

In the second example, the code is both functional and "literal", aka we can read it as text.

Reply:
I'm full on for literal

Reply:
Using underscores for emphasis is not a markdown innovation either: https://en.wikipedia.org/wiki/Emphasis_(typography)#Punctuation_marks